openedx/core/lib/studio_entities.py

```diff
@@ -0,0 +1,64 @@
+"""Studio entity plugin manager and API."""
+import abc
+
+from openedx.core.lib.api.plugins import PluginManager
+
+
+class StudioEntityPluginManager(PluginManager):
```

Comment:
Course tabs are used in both Studio and LMS so it makes sense for them to go in openedx. Since these are Studio only extensions, I recommend putting the code somewhere in Studio too.